__tests__/src/elementAXObjectMap-test.js

```diff
@@ -3,9 +3,423 @@ import expect from 'expect';
+            [{"name": "footer"}, ["FooterRole"]],
+            [{"name": "form"}, ["FormRole"]],
+            [{"name": "h1"}, ["HeadingRole"]],
+            [{"name": "h2"}, ["HeadingRole"]],
+            [{"name": "h3"}, ["HeadingRole"]],
+            [{"name": "h4"}, ["HeadingRole"]],
+            [{"name": "h5"}, ["HeadingRole"]],
+            [{"name": "h6"}, ["HeadingRole"]],
+            [{"name": "iframe"}, ["IframeRole"]],
+            [{"attributes": [{"name": "usemap"}], "name": "img"}, ["ImageMapRole"]],
+            [{"name": "img"}, ["ImageRole"]],
+            [{"name": "input"}, ["InlineTextBoxRole", "TextFieldRole"]],
+            [{"attributes": [{"name": "type", "value": "time"}], "name": "input"}, ["InputTimeRole"]],
+            [{"name": "label"}, ["LabelRole"]],
+            [{"name": "legend"}, ["LegendRole"]],
+            [{"name": "br"}, ["LineBreakRole"]],
+            [{"attributes": [{"name": "href"}], "name": "a"}, ["LinkRole"]],
+            [{"name": "option"}, ["ListBoxOptionRole"]],
+            [{"name": "datalist"}, ["ListBoxRole"]],
+            [{"name": "select"}, ["ComboBoxRole", "ListBoxRole"]],
+            [{"name": "li"}, ["ListItemRole"]],
+            [{"name": "ul"}, ["ListRole"]],
+            [{"name": "ol"}, ["ListRole"]],
+            [{"name": "main"}, ["MainRole"]],
+            [{"name": "mark"}, ["MarkRole"]],
+            [{"name": "marquee"}, ["MarqueeRole"]],
+            [{"name": "menuitem"}, ["MenuItemRole"]],
+            [{"name": "menu"}, ["MenuRole"]],
+            [{"name": "meter"}, ["MeterRole"]],
+            [{"name": "nav"}, ["NavigationRole"]],
+            [{"name": "p"}, ["ParagraphRole"]],
+            [{"name": "pre"}, ["PreRole"]],
+            [{"name": "progress"}, ["ProgressIndicatorRole"]],
+            [{"attributes": [{"name": "type", "value": "radio"}], "name": "input"}, ["RadioButtonRole"]],
+            [{"attributes": [{"name": "scope", "value": "row"}], "name": "th"}, ["RowHeaderRole"]],
+            [{"name": "tr"}, ["RowRole"]],
+            [{"name": "ruby"}, ["RubyRole"]],
+            [{"attributes": [{"name": "type", "value": "search"}], "name": "input"}, ["SearchBoxRole"]],
+            [{"attributes": [{"name": "type", "value": "range"}], "name": "input"}, ["SliderRole"]],
+            [{"attributes": [{"name": "type", "value": "number"}], "name": "input"}, ["SpinButtonRole"]],
+            [{"attributes": [{"name": "type", "value": "checkbox"}], "name": "input"}, ["CheckBoxRole", "SwitchRole"]],
+            [{"name": "table"}, ["TableRole"]],
+            [{"name": "input"}, ["InlineTextBoxRole", "TextFieldRole"]],
+            [{"attributes": [{"name": "type", "value": "text"}], "name": "input"}, ["TextFieldRole"]],
+            [{"name": "time"}, ["TimeRole"]],
+            [{"name": "video"}, ["VideoRole"]],
+          ]
+        )
+      );
+      expect([...elementAXObjectMap.entries()]).toEqual(
```

Comment:
@ljharb so for instance, the entries, values and keys methods can still be iterator, such as with the spread operator or the for..of control.